invenio_cli/helpers/cli_config.py

```diff
@@ -48,6 +48,22 @@ def update_instance_path(self, new_instance_path):
         with open(self.fullpath, 'w') as configfile:
             self.config.write(configfile)
 
+    def get_services_setup(self):
+        """Returns bool whether services have been setup or not."""
+        return self.config.getboolean(CLIConfig.CLI_SECTION, 'services_setup')
+
+    def update_services_setup(self, is_setup):
+        """Updates path to application instance directory."""
+        assert isinstance(is_setup, bool)
```

Comment:
doubt: in which case it would not be a boolean? I am just not a fan of asserts in the code

Reply:
> not a fan of asserts in the code

I hear you. If we were ever considering adding Python types to the code base, this would be a valuable place to put some.

... After thinking about it, we can remove it. On retrieval we use configparser's `getboolean` which will raise an Error if the value is not a boolean equivalent string. This way, even if someone passes a non-boolean (Python is dynamic, other things than boolean might be passed), on retrieval an error will be raised and the source of the problem will be identified.
Removing!